### ISSUE TITLE
Sprite icons still not working

### ISSUE THREAD

**Issue by tmaier** [vwall/compass-twitter-bootstrap#32]
[_variables.scss#L58-59](https://github.com/vwall/compass-twitter-bootstrap/blob/master/stylesheets/compass_twitter_bootstrap/_variables.scss#L58-59) defines a wrong path to the icons

``` scss
// Sprite icons path
$iconSpritePath:          "../img/glyphicons-halflings.png" !default;
$iconWhiteSpritePath:     "../img/glyphicons-halflings-white.png" !default;
```

**Comment by jpfuentes2** [comment]
Because this is now using the compass `image-url` helper, this shouldn't include a path and instead be the basename version of the file. That way the compass.rb config or options passed to executable can set the image directory path.

**Comment by vwall** [comment]
Fixed in 1c011945827474fea30f68366a65beef111e7f98

**Comment by bopm** [comment]
This error introduced again with upgrade to 2.0.3.
